signalfx-tracing/signalfx-python-tracing/django/blog/posts/tests.py

```diff
@@ -0,0 +1,25 @@
+from django.test import TestCase
```

Comment:
Is this file necessary?

Reply:
That was part of my .gitignore file as well. Need to remove a few files

Reply:
Added to .gitignore and removed